Assets/Scripts/ARObjectSpawner.cs

```diff
@@ -77,18 +77,16 @@ private void UpdateSpawned(ARTrackedImage image)
         if (image.trackingState == TrackingState.Tracking)
         {
             spawned.gameObject.transform.position = image.transform.position;
-            spawned.gameObject.transform.position += new Vector3(-0.025f, 0, -0.1f);
-
-            if (image.transform.rotation.eulerAngles.x > 60 || image.transform.rotation.eulerAngles.z > 60 ||
-                image.transform.rotation.eulerAngles.x < -60 || image.transform.rotation.eulerAngles.z < -60)
+            if (name == "floor_plan2")
             {
-                spawned.gameObject.transform.rotation = image.transform.rotation;
-            }
-            else
+                spawned.gameObject.transform.rotation = Quaternion.Euler(new Vector3(0, 180f, 0));
+            }if (name == "floor_plan1")
             {
-                spawned.gameObject.transform.rotation = Quaternion.Euler(0, image.transform.rotation.eulerAngles.y, 0);
+                Debug.Log(image.transform.position);
+                spawned.gameObject.transform.rotation = Quaternion.Euler(new Vector3(0, 20f, 0));
             }
 
+
```

Comment:
I don't understand why this change is needed, the object should rotate according to the image.

And you corrected the angle according to the object, can't you change the default orientation of the prefab itself?